src/containers/shared/components/DomainLink.tsx

```diff
@@ -7,14 +7,27 @@ export interface Props {
   domain: string
 }
 
+// Matches a protocol (e.g. 'http://' or 'https://') at the start of a string.
+const protocolRegex = /^([a-z][a-z0-9+\-.]*):\/\//
+
 const DomainLink = (props: Props) => {
   const { className, decode = false, domain } = props
+
+  // If decode is true, decode the domain
+  const decodedDomain = decode ? decodeHex(domain) : domain
+
+  // Use the test method to check for the protocol
+  const domainHasProtocol = protocolRegex.test(decodedDomain)
+
+  // If decoded domain does not have a protocol, add one ; otherwise, don't
+  const href = domainHasProtocol ? decodedDomain : `https://${decodedDomain}`
```

Comment:
@ckniffen should this be `http://` or `https://`? What if a domain doesn't have HTTPS enabled?